scheduler/lib/models/scheduler.proto

```diff
@@ -33,10 +33,10 @@ message Target {
     // Name of the scheduler to trigger this target.
     // Defaults to being triggered by cocoon.
     optional SchedulerSystem scheduler = 8 [default = cocoon];
-    // Whether target should run pre-submit.
-    optional bool presubmit = 9 [default = true];
-    // Whether target should run post-submit.
-    optional bool postsubmit = 10 [default = true];
+    // Whether target should run pre-submit. Defaults to false, will not run presubmit.
+    optional bool presubmit = 9 [default = false];
```

Comment:
Won't this lead to an overall more verbose `.ci.yaml` file, once most of the targets are running post and pre?

Reply:
Good point, I was looking at this from the cocoon perspective. I swapped backed the default=true to better accomodate the framework

Reply:
This LGTM.

As an aside, once we have the whole flutter world using this proto for configuration, I foresee that there will be a lot of things we'll want to change but can't easily do without breaking everything. Down the line, we may want to think about versioning the `.ci.yaml` files, so we can rev the protobuf with breaking changes.